mod/wikirate/set/type_plus_right/metric_value/source.rb

```diff
@@ -2,7 +2,7 @@
   view :editor do |args|
     
     if card.new?
-      source = Card.new :type_code=>:source
+      source = Card.new type_code: :source, name: "new source"
```

Comment:
Newing a card with exiting card name should not cause any problem because it will not cause conflict until we really save it.